src/views/support.njk

```diff
@@ -0,0 +1,122 @@
+{% extends "base.njk" %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% block pageTitle %}Support – GOV.UK Product Pages{% endblock %}
+
+{% set mainClasses = "govuk-main-wrapper govuk-main-wrapper--auto-spacing" %}
+
+{% block beforeContent %}
+  <div class="govuk-width-container">
+    <div class="govuk-breadcrumbs">
+      <ol class="govuk-breadcrumbs__list">
+        <li class="govuk-breadcrumbs__list-item">
+          <a class="govuk-breadcrumbs__link" href="https://www.gov.uk/service-toolkit#gov-uk-services">GOV.UK services</a>
+        </li>
+        <li class="govuk-breadcrumbs__list-item">
+          <a class="govuk-breadcrumbs__link" href="/">GOV.UK Sign In</a>
+        </li>
+        <li class="govuk-breadcrumbs__list-item">
+          <a class="govuk-breadcrumbs__link"  href="#main-content" aria-current="page">Support</a>
+        </li>
+      </ol>
+    </div>
+  </div>
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-width-container">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        {% if valueNotSelected %}
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: [
+            {
+              text: "You must select an option to tell us what you need help with",
+              href: "#support-error"
+            }
+          ]
+        }) }}
+        {% endif %}
+
+        <h1 class="govuk-heading-l">Support</h1>
+
+        <form action="/support" method="post" novalidate>
+
+          {% if valueNotSelected %}
+
+            {{ govukRadios({
+              idPrefix: "support",
+              name: "support",
+              fieldset: {
+                legend: {
+                  text: "What do you need help with?"
+                }
+              },
+              errorMessage: {
```

Comment:
Does this work if we wrap the error message in the if statement?  I think that's the only part that differs between each block isn't it?  If that's so, we can avoid repeating the items code.

Reply:
We can not make the "errorMessage: {  text: "You must select an option to tell us what you need help with"}"